.github/workflows/release.yaml

```diff
@@ -0,0 +1,58 @@
+name: Release on pushed tags
+
+on:
+  push:
+    tags:
+      - '*'
+  # TODO: remove
+  pull_request:
+
+jobs:
+  release:
+    runs-on: ubuntu-22.04
+    # TODO: remove
+    env:
+      tag: test.0.9.2
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      # TODO: uncomment
+      # - name: Get tag name
+      #   run: echo "tag=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+
+      - name: Update env.sample
+        run: |
+          cp docker/env.sample ${{ runner.temp }}
+          sed -i 's/^TARANIS_TAG=.*/TARANIS_TAG=${{ env.tag }}/' ${{ runner.temp }}/env.sample
+      
+      - name: Create tar.gz of repository
+        uses: a7ul/tar-action@v1.2.0
+        id: compress
+        with:
+          command: c
+          files: .
+          outPath: ${{ runner.temp }}/${{ env.tag }}.tar.gz
+
+      - name: Create zip of repository
+        run: zip -r ${{ runner.temp }}/${{ env.tag }}.zip .
+
```

Comment:
Tags always include a tar of the code so this is obsolete, and should be removed